R/get_national_data.R

```diff
@@ -44,9 +44,15 @@ get_national_data <- function(country, source = "who", steps = FALSE,
   nation_class$download()
 
   # dataset specifc cleaning
+  if (verbose) {
+    message(paste("- Cleaning", source, "data"))
```

Comment:
can these not live in the internal functions themselves? (perhaps as non-exported functions to save duplication). That way places can have custom messages?